src/Control/Email/SwiftMailer.php

```diff
@@ -9,6 +9,8 @@
 use Swift_Message;
 
 /**
+ * @deprecated 4.12.0 Will be replaced with symfony/mailer
```

Comment:
Should we also add a constructor with a `Deprecation::notice()`?

Reply:
Possibly, though I'm not sure.

I think we leave this as is for now and tackle globally in https://github.com/silverstripe/silverstripe-framework/issues/10500

Reply:
I've added a note to that card so that it doesn't get forgotten. That's sufficient for now.